packs/slack/actions/send_invite.yaml

```diff
@@ -0,0 +1,26 @@
+---
+  name: "send_invite"
+  runner_type: "run-python"
+  description: "Send an invitation to join a Slack Org"
+  enabled: true
+  entry_point: "send_invite.py"
+  parameters:
+    email:
+      type: "string"
+      description: "Email address to send invitation to."
+      required: true
+    first_name:
+      type: "string"
+      description: "Users first name"
+    channels:
+      type: "string"
```

Comment:
This should be an array and we should build a comma delimited string inside an action.